src/models/accounts.rs

```diff
@@ -4,18 +4,21 @@ use schema::accounts;
 #[derive(Insertable, Debug, Clone)]
 pub struct Account {
     pub account_id: String,
-    pub created_by_receipt_id: String,
+    pub created_by_receipt_id: Option<String>,
     pub deleted_by_receipt_id: Option<String>,
 }
 
 impl Account {
     pub fn new(
         account_id: String,
-        created_by_receipt_id: &near_indexer::near_primitives::hash::CryptoHash,
+        created_by_receipt_id: Option<&near_indexer::near_primitives::hash::CryptoHash>,
```

Comment:
I suggest we have two methods: `new_from_genesis` (only `account_id` is required) and `new_from_receipt` (`account_id` and `created_by_receipt_id` is required).